.packit.yaml

```diff
@@ -170,6 +170,17 @@ jobs:
     fmf_url: "https://gitlab.cee.redhat.com/baseos-qe/integration_scripts.git"
     tmt_plan: '/tmt/integration/plan'
 
+  # Run missing tests identified in Connect
+  - <<: *test-base
+    <<: *internal
+    <<: *require-full-tests
+    identifier: internal-fusa
+    fmf_url: "https://gitlab.cee.redhat.com/automotive/tests/all.git"
+    fmf_ref: "main"
```

Comment:
The default branch, in this case `main`, does not have to be specified, it is used by default. So you can drop this line.

Reply:
Btw, if you have a git branch where it is working, you could make a temporary commit pointing to it instead so that we can check the status for it.